library/haskell

```diff
@@ -1,5 +1,6 @@
 Maintainers: Peter Salvatore <peter@psftw.com> (@psftw),
              Christopher Biscardi <chris@christopherbiscardi.com> (@ChristopherBiscardi)
```

Comment:
These values are comma-separated, so this line needs a comma: :sweat_smile:

```suggestion
             Christopher Biscardi <chris@christopherbiscardi.com> (@ChristopherBiscardi),
```

(Also, welcome! :tada:)

cc @psftw @ChristopherBiscardi (FYI)

Reply:
yikes... fixed :-)